### ISSUE TITLE
Question: how to aggregate multiple `prompts.txt`

### ISSUE THREAD

**Issue by jamesbraza** [promptfoo/promptfoo#170]
I have a few fundamentally different conversations I want to test, let's say 3 for simplicity.  So thus I have three different `prompts.txt` files.  Each `prompts.txt` file has its own unique set of variables and tests.

How can one either:
- Have one `promptfooconfig` that is able to have multiple (`prompts.txt`, set of `vars`, set of `assert`s)
- Aggregate the results of 2+ `promptfooconfig` from different subdirectories?

**Comment by jamesbraza** [comment]
So I see for the [`promptfoo-action`](https://promptfoo.dev/docs/integrations/github-action#using-the-github-action), it's looking in `prompts/`.  What does the directory structure within `prompts/` look like?  Are there like subfolders with many pairs of `prompts.txt`/`promptfooconfig.yaml`s?

```txt
prompts
├── can-fly
│   ├── promptfooconfig.yaml
│   └── prompts.txt
└── can-swim
    ├── promptfooconfig.yaml
    └── prompts.txt
```

**Comment by typpo** [comment]
Is the desired result to have all the vars and tests combined into a single view?  In this case, would it only make sense if there was some overlap between the vars/prompts/models being tested?

**Comment by jamesbraza** [comment]
I think it's combining the results of 2+ configs (with separate vars/prompts/tests) into:

- Main ask: aggregated statistics
- Nice to have: one `promptfoo view`

So lets say `can-fly` will have 10 passes and 2 failures, and `can-swim` will have 6 passes and 4 failures.  How can those be combined into a summary statistic (and possibly one `promptfoo view`), with just one `promptfoo` invocation?

In other words, it's like having `promptfoo` run on an entire directory of configs, and report an aggregated result

**Comment by typpo** [comment]
Got it.  I think https://github.com/promptfoo/promptfoo/pull/285 should solve this use case, both aggregated stats and aggregated view.  The web view might get a bit unwieldy if the evals don't have prompt/provider/var overlap.

**Comment by jamesbraza** [comment]
Oh nice I like that, sweet! Can you add the ability to crawl directories?  Something like `promptfoo eval -d prompts`

**Comment by typpo** [comment]
Added :)  Will land the PR shortly

**Comment by jamesbraza** [comment]
Looks like the PR landed, closing this out, thank you again!
